test/e2e/cron/basic-update.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Not a fan of creating a new "`-update`" file for the necessary update, but it seems to be the most robust way to do the test and matches what the user experience would be. The alternative would be editing the original file before "updating" it